man/systemd.network.xml

```diff
@@ -153,6 +153,24 @@
             </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>SSID=</varname></term>
+          <listitem>
+            <para>A whitespace-separated list of shell-style globs matching the SSID of the currently
+            connected wireless network. If the list is prefixed with a "!", the test is inverted.
```

Comment:
Not anything wrong but do we have to explicitly mention the '!' thing ?

Reply:
OK, but I'd like to that in other PR, as we should update all other entries in [Match] section.